docs/multi_agent.md

```
@@ -7,31 +7,23 @@ Orchestration refers to the flow of agents in your app. Which agents run, in wha
 
 You can mix and match these patterns. Each has their own tradeoffs, described below.
 
-## Orchestrating via LLM
+We have a number of examples in examples/cai/agent_patterns.
 
-An agent is an LLM equipped with instructions, tools and handoffs. This means that given an open-ended task, the LLM can autonomously plan how it will tackle the task, using tools to take actions and acquire data, and using handoffs to delegate tasks to sub-agents. For example, a research agent could be equipped with tools like:
+#### ◉ Orchestrating via LLM
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)